template/templates/views/feed_entries.html

```diff
@@ -27,6 +27,9 @@ <h1 dir="auto">
             <a href="{{ route "feedEntries" "feedID" .feed.ID }}">{{ t "menu.show_only_unread_entries" }}</a>
         </li>
         {{ end }}
+        <li>
+            <a href="{{ route "markFeedAsRead" "feedID" .feed.ID }}?redirect_to_self">{{ t "menu.mark_all_as_read" }}</a>
```

Comment:
To be consistent with other pages, it would be better to move the link next to "mark this page as read". Line 19, inside the `if` statement.

Reply:
Indeed, good point, I 'll amend.